README.md

```diff
@@ -58,8 +58,8 @@ There's also [French documentation](http://blog.valouille.fr/2014/03/creer-un-de
 
 ## Contribute
 
-Freight is [BSD-licensed](https://github.com/rcrowley/freight/blob/master/LICENSE)
+Freight is [BSD-licensed](https://github.com/freight-team/freight/blob/master/LICENSE)
 
-* Source code: <https://github.com/rcrowley/freight>
-* Issue tracker: <https://github.com/rcrowley/freight/issues>
-* Wiki: <https://github.com/rcrowley/freight/wiki>
+* Source code: <https://github.com/freight-team/freight>
+* Issue tracker: <https://github.com/freight-team/freight/issues>
```

Comment:
This is disabled on the repo right now, you'll need to go to Settings and enable it.